doc/cephfs/multimds.rst

```diff
@@ -248,42 +248,45 @@ Dynamic Subtree Partitioning
 
 CephFS has long had a dynamic metadata balancer (sometimes called the "default
 balancer") which can split or merge subtrees while placing them on "colder" MDS
-ranks. Moving the metadata around can improve overall file system throughput
+ranks. Moving the metadata in this way improves overall file system throughput
 and cache size.
 
-However, the balancer has suffered from problem with efficiency and performance
-so it is by default turned off. This is to avoid an administrator "turning on
-multimds" by increasing the ``max_mds`` setting and then finding the balancer
-has made a mess of the cluster performance (reverting is straightforward but
-can take time).
+However, the balancer is sometimes insufficiently efficient and insufficiently
```

Comment:
However, the balancer is sometimes inefficient or slow, so by default it is disabled.

Reply:
Yeah, okay.